recipe/meta.yaml

```diff
@@ -53,7 +53,7 @@ build:
   skip: True  # [py<37]
```

Comment:
either you should disable for now python 3.11, or add the numpy 1.22,<2 dependency for it to the recipe.